src/ethereum_test_tools/spec/state/types.py

```diff
@@ -44,49 +43,63 @@ class FixtureEnvironment:
         default=100000000000000000,
         json_encoder=JSONEncoder.Field(
             name="currentGasLimit",
-            cast_type=Number,
+            cast_type=ZeroPaddedHexNumber,
         ),
     )
     number: NumberConvertible = field(
         default=1,
         json_encoder=JSONEncoder.Field(
             name="currentNumber",
-            cast_type=Number,
+            cast_type=ZeroPaddedHexNumber,
         ),
     )
     timestamp: NumberConvertible = field(
         default=1000,
         json_encoder=JSONEncoder.Field(
             name="currentTimestamp",
-            cast_type=Number,
+            cast_type=ZeroPaddedHexNumber,
         ),
     )
     prev_randao: Optional[NumberConvertible] = field(
         default=None,
         json_encoder=JSONEncoder.Field(
             name="currentRandom",
-            cast_type=Number,
+            cast_type=Hash,
         ),
     )
     difficulty: Optional[NumberConvertible] = field(
         default=None,
         json_encoder=JSONEncoder.Field(
             name="currentDifficulty",
-            cast_type=Number,
+            cast_type=ZeroPaddedHexNumber,
         ),
     )
     base_fee: Optional[NumberConvertible] = field(
         default=None,
         json_encoder=JSONEncoder.Field(
             name="currentBaseFee",
-            cast_type=Number,
+            cast_type=ZeroPaddedHexNumber,
         ),
     )
     excess_blob_gas: Optional[NumberConvertible] = field(
         default=None,
         json_encoder=JSONEncoder.Field(
-            name="currentExcessBlobGas",
-            cast_type=Number,
+            name="parentExcessBlobGas",
+            cast_type=ZeroPaddedHexNumber,
+        ),
+    )
+    beacon_root: Optional[FixedSizeBytesConvertible] = field(
+        default=None,
+        json_encoder=JSONEncoder.Field(
+            name="currentBeaconRoot",
+            cast_type=Hash,
+        ),
+    )
```

Comment:
I think we need to discuss this a bit further because this is a block property, so it requires the state test consumers to perform a block action before executing the transactions, while we could simply put this result in the state (just like withdrawals for example).